vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj.filters

```diff
@@ -712,6 +817,7 @@
     <Midl Include="JsiApi.idl" />
     <Midl Include="DocString.idl" />
     <Midl Include="DesktopWindowMessage.idl" />
+    <Midl Include="ReactCoreInjection.idl" />
```

Comment:
I think the changes to the vcxproj.filters file were not meant to be committed  :)